src/Cdn/Cdn.Autorest/examples/Enable-AzFrontDoorCdnProfileMigration.md

```diff
@@ -1,13 +1,13 @@
-### Example 1: Commit the migrated Azure Front Door(Standard/Premium) profile.
+### Example 1: Enable an AzureCDN custom domain under the AzureCDN endpoint
 ```powershell
-Enable-AzFrontDoorCdnProfileMigration -ProfileName name-migrated -ResourceGroupName rgName
+$customDomainHttpsParameter = New-AzCdnManagedHttpsParametersObject -CertificateSourceParameterCertificateType Dedicated -CertificateSource Cdn  -ProtocolType TLS12
+Enable-AzCdnCustomDomainCustomHttps -ResourceGroupName testps-rg-da16jm -ProfileName cdn001 -EndpointName endptest001 -CustomDomainName customdomain001 -CustomDomainHttpsParameter $customDomainHttpsParameter
```

Comment:
Please fix the example. This the example for `Enable-AzCdnCustomDomainCustomHttps`